### ISSUE TITLE
move project to QGIS core

### ISSUE THREAD

**Issue by tomkralidis** [geopython/MetaSearch#62]
cc @geopython/metasearch 

The team has decided to move this project into QGIS core.  Given that MetaSearch is now a core QGIS plugin, it will be easier to maintain in the long run inline with QGIS, especially for mainline documentation, translation and issue tracking.

Plan:
- [x] create http://hub.qgis.org/projects/metasearch
- [x] move wiki pages and post redirect on geopython/MetaSearch wiki, turn off
- [x] port GitHub issues to qgis issue tracker
- [x] move codebase (we still want to deploy independently and there are many files in the current root directory as part of the move)
- [x] move docs to QGIS-Documentation
- [x] redirect http://geopython.github.io/MetaSearch to http://hub.qgis.org/projects/MetaSearch
- [x] integrate and turn off MetaSearch [Transifex](https://www.transifex.com/projects/p/MetaSearch) (N.B. there is a Polish translation in QGIS that is NOT in MetaSearch, be sure not to overwrite/lose it)
- [x] turn off this issue tracker

**Comment by tomkralidis** [comment]
@rduivenvoorde any suggestions on how / where do we move docs and their translations into QGIS proper? Does this go into https://github.com/QGIS/QGIS-Documentation?

**Comment by rduivenvoorde** [comment]
@tomkralidis I've managed to download the documentation po files from MetaSearch Transifex and upload them to the QGIS-Documentation project for all languages at MetaSearch Transifex.

For the templates/plugin translations, it seemed that in the mean time all languages already translated that theirselves again, so I decided NOT to override those translations.
I sent an email to the coordinators of the MetaSearch Transifex to tell them they could have a look and maybe copy translations from MetaSearch tx, but that the project will be closed in short time.

So to me the 'transifex' move is done
